src/main/java/com/paneedah/mwc/items/guns/Remington870Factory.java

```diff
@@ -21,8 +21,8 @@ public Item createGun(CommonProxy commonProxy) {
 
         .withName("remington870")
 //      .withAmmo(CommonProxy.Remington870Mag)
-        .withAmmoCapacity(4)
-        .withMaxBulletsPerReload(4)
+        .withAmmoCapacity(5)
+        .withMaxBulletsPerReload(5)
```

Comment:
This was correct, so please revert this change.
Refer to https://www.remarms.com/shotguns/pump-action/model-870/